sys/kern/main.c

```diff
@@ -40,6 +40,19 @@ static void mount_fs(void) {
   do_fchmodat(p, AT_FDCWD, "/tmp", ACCESSPERMS | S_ISTXT, 0);
 }
 
+static void open_fds(void) {
```

Comment:
Since that code was moved to #1390 we can safely remove it from this PR.

Reply:
Reverted all the changes in `sys/kern.main.c` in https://github.com/cahirwpz/mimiker/pull/1371/commits/ac3245cf093932672daac3f36f3689a38e062b11